units/unit.cpp

```diff
@@ -23,6 +23,7 @@ Unit::Unit(const std::string &unit)
 std::string Unit::name() const {
   auto repr = to_string(m_unit);
   repr = std::regex_replace(repr, std::regex("^u"), "µ");
+  repr = std::regex_replace(repr, std::regex("count"), "counts");
```

Comment:
Are there cases where `"count"` or `"item"` are surrounded by other word characters? I.e. embedded in another name of a unit?

If the underlying library ever produces `"counts`", the replacement turns it into `"countss"`.